tests/api_tests/3000/api_profile_test.spec.ts

```diff
@@ -39,6 +39,12 @@ test.describe('Profile API test', async () => {
         await api.leadersPage.getLeders(apiUrl.qaEnvUrl, user.userToken, 'month')
     })
 
+    test('Show Month Leader Name', async () => {
```

Comment:
@PikachuSerg1 as I mention earlier, remove this test and modify existing ones

Reply:
Updated